background.js

```diff
@@ -1,15 +1,14 @@
 function onBeforeRequest(data){
     //perhaps put 'data' in parameter
     console.log("top of on before request method");
-    console.log(data.initiator);
+    console.log(data.url);
     //need to return something
     //this is where blocking goes and also figuring out whether something needs to be blocked
 
     //rework all of this!
-    var urlNotFixed = data.initiator;
-    urlNotFixed.replace(" ", "");
+    const urlNotFixed = data.url;
     urlNotFixed.toLowerCase();
-    var urlFixed = urlNotFixed;
+    const urlFixed = urlNotFixed;
```

Comment:
This is not gonna work as you probably expect, and to understand why, look closely at the Syntax and Description sections of [the docs for `String.prototype.toLowerCase()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase).

To get the behavior you probably intend, the easiest thing to do is to dispense with `urlNotFixed` altogether and consolidate the expressions in line 9 and 10 into a single expression that calls `toLowerCase` directly on the `url` property of `data`, then assign the return value of that expression to `urlFixed`.